src/test/store/receive-profile.test.js

```diff
@@ -1269,16 +1269,34 @@ describe('actions/receive-profile', function() {
       ).toMatchSnapshot();
     });
 
-    // eslint-disable-next-line jest/no-disabled-tests
-    it.skip('can load gzipped json', async function() {
+    it('can load gzipped json', async function() {
       // TODO - See issue #1023. The zee-worker is failing to compress/decompress
```

Comment:
Why are these TODOs here still? We are fixing them in this PR :)

Reply:
yea :), i'll take it out